src/graphql/admin/root/mutation/account-update-level.ts

```diff
@@ -0,0 +1,46 @@
+import { updateAccountLevel } from "@app/accounts/update-account-level"
+import AccountDetailPayload from "@graphql/admin/types/payload/account-detail"
+import AccountLevel from "@graphql/admin/types/scalar/account-level"
+import { mapError } from "@graphql/error-map"
+import { GT } from "@graphql/index"
+
+const AccountUpdateLevelInput = new GT.Input({
+  name: "AccountUpdateLevelInput",
+  fields: () => ({
+    uid: {
+      type: GT.NonNullID,
+    },
+    level: {
+      type: GT.NonNull(AccountLevel),
+    },
+  }),
+})
+
+const AccountUpdateLevelMutation = GT.Field({
+  type: GT.NonNull(AccountDetailPayload),
+  args: {
+    input: { type: GT.NonNull(AccountUpdateLevelInput) },
+  },
+  resolve: async (_, args) => {
+    const { uid, level } = args.input
+
+    // FIXME: nicolas: I don't understand the purpose of this below
+    for (const input of [uid, level]) {
```

Comment:
@samerbuna is this that uid and level are being checked by graphql with `parseLiteral` and/or `parseValue`?

Reply:
They are not, but we're generically checking all inputs here in case in the future graphql parsing checks are added.

Reply:
why having this code if this is a no-op for the time being? isn't better to add the check when we are actually using the graphql check? otherwise it may make developer thinks/mislead the inputs are actually already checked.

Reply:
Fair point. We should really leverage TS types here to figure out which inputs are checked. I'll look into the best way to do that.